### PR TITLE
added user and app models

### PR DESCRIPTION
**Pull request by young-k**

### CODE REVIEW

**Comment by Jma353** [comment]
Overall comment - 2 spaces on indenting, not 4.

File: src/app/events/models/user.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import bcrypt
+from . import *
+
+class User(Base):
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(255), nullable=False)
+    password = db.Column(db.Text, nullable=False)
```
Comment (Jma353) [inline src/app/events/models/user.py]: `password_digest` instead of `password` - since you hash / salt.

File: src/app/events/models/app.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from . import *
+
+class App(Base):
+    __tablename__ = 'apps'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False)
+    secret_key = db.Column(db.String(255), unique=True, nullable=False)
+    event_types = db.relationship('EventType', backref='app')
```
Comment (Jma353) [inline src/app/events/models/app.py]: Talk to @noobbyte about this - this relationship should be defined in `event_type` model tbh.

File: src/app/events/models/user.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import bcrypt
+from . import *
+
+class User(Base):
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(255), nullable=False)
```
Comment (Jma353) [inline src/app/events/models/user.py]: Uniqueness constraint too

**Comment by Jma353** [comment]
Also, a suggestion for resting these change, migrate the db + screenshot the resultant schema.  You can also import the flask app into python repl and manually create models to make sure they are being stored properly.

**Comment by Jma353** [comment]
@young-k screenshot or evidence of proper migration of the db?

**Comment by young-k** [comment]
screen shot 2017-09-25 at 9 44 02 am
^ screenshot of the the `upgrade` function of the `revision` in the `migrations` directory

**Comment by Jma353** [comment]
screen shot 2017-09-25 at 1 55 04 pm

**Comment by young-k** [comment]
merge sha: 6ea49b92788ede3c03d15b18fb1e5571dad65c43

**Pull request merged**